rdflib/graph.py

```diff
@@ -968,6 +967,12 @@ def qname(self, uri):
     def compute_qname(self, uri, generate=True):
         return self.namespace_manager.compute_qname(uri, generate)
 
+    def qname_strict(self, uri):
```

Comment:
Surely the move here is just to reach consistency of API? If you, @mwatts15, want a change in usage patterns, perhaps after this PR you would have to address the existing `qname()` & ``compute_qname()` as well as  `qname_strict`?

So are you saying you use a stand-alone Namespace Manager object do do all your namespaceing, perhaps like:

```python
from rdflib.namespace import NamespaceManager

nm = NamespaceManager()

g = Graph()
g.namespace_manager(nm)

other_gr = Graph()
other_g.namespace_manager(nm)

# etc.
```

Reply:
> Why add more wrappers to namespace_manager? I don't see any additional usages in this PR.

The other part of this commit details the issue resolved, when serialized as xml an Exception is raised by the XML serialiser: `# 'Dataset' object has no attribute 'compute_qname_strict'`. I'm not minded to _prohibit_ serialisation of a Dataset as triples in XML (albeit, only the triples in the default graph get output) and, given that nicholascar and ashleysommer are considering a ConjunctiveGraph/Dataset dedupe, I thought this approach was the least impactful.

Reply:
> @gjhiggins, from where is `'Dataset' object has no attribute 'compute_qname_strict'` reported? I would think the solution is just to pass `dataset.namespace_manager` instead of `dataset` in such a case.

In [this test](https://github.com/RDFLib/rdflib/blob/1ed5a13df82b25af28f3843e15725f1e4908e8ee/test/test_identifier_as_context/test_dataset_default_graph.py#L132) of serializations of the Dataset default graph.

```py
dataset_default_graph = ds.get_context(DATASET_DEFAULT_GRAPH_ID)
dataset_default_graph.serialize(format="xml")  # <-- results in the Exception
```
Apologies for mistakenly stating earlier that it was a Dataset issue, not a Dataset default graph issue.